core/server/worker/src/test/java/alluxio/worker/page/PagedBlockWriterTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.page;
+
+import static org.junit.Assert.assertEquals;
+
+import alluxio.client.file.cache.CacheManager;
+import alluxio.client.file.cache.DefaultPageMetaStore;
+import alluxio.client.file.cache.LocalCacheManager;
+import alluxio.client.file.cache.PageId;
+import alluxio.client.file.cache.PageMetaStore;
+import alluxio.client.file.cache.PageStore;
+import alluxio.client.file.cache.evictor.CacheEvictor;
+import alluxio.client.file.cache.evictor.FIFOCacheEvictor;
+import alluxio.client.file.cache.store.LocalPageStoreDir;
+import alluxio.client.file.cache.store.LocalPageStoreOptions;
+import alluxio.client.file.cache.store.PageStoreOptions;
+import alluxio.conf.Configuration;
+import alluxio.conf.InstancedConfiguration;
+import alluxio.conf.PropertyKey;
+import alluxio.util.CommonUtils;
+import alluxio.util.WaitForOptions;
+import alluxio.util.io.BufferUtils;
+
+import com.google.common.collect.ImmutableList;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+
+public class PagedBlockWriterTest {
+  private static final int TEST_BLOCK_SIZE = 1024;
+  private static final int TEST_PAGE_SIZE = 128;
```

Comment:
Test with more page sizes? the test fails with page size = 129, but passes with 127.